lib/deck.rb

```diff
@@ -19,4 +19,12 @@ def cards_in_category(category)
     matching_cards
    end
 
```

Comment:
This is a method that I manually implemented into the Runner file. It would work off of Deck if I can figure out a good way to implement a non-destructive way to index the Turns.